### PR TITLE
Export more things for the extension API

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

We need to export these things (and others as we find them) because `@k8slens/extensions` is now a separate package and the older `@k8slens/extensions/dist/src/...` type exports (which we never officially supported anyway) no longer works.

Future work: 
- Turn on `requiredToBeDocumented`
- Turn on `treatWarningsAsErrors`
- Turn on `validation`

### CODE REVIEW

File: packages/core/src/extensions/common-api/utils.ts [hunk 1]
```diff
@@ -7,20 +7,23 @@ import openLinkInBrowserInjectable from "../../common/utils/open-link-in-browser
...
-  openBrowser: asLegacyGlobalFunctionForExtensionApi(openLinkInBrowserInjectable),
-  getAppVersion: () => {
-    const di = getLegacyGlobalDiForExtensionApi();
-
-    return di.inject(buildVersionInjectable).get();
-  },
-} as const;
+
+export { Singleton } from "../../common/utils";
+export { prevDefault, stopPropagation } from "../../renderer/utils/prevDefault";
```
Comment (jim-docker) [inline packages/core/src/extensions/common-api/utils.ts:12]: no longer exported as part of `Util`?

File: .github/workflows/publish-release-npm.yml [hunk 2]
```diff
@@ -38,7 +38,9 @@ jobs:
       - name: Publish NPM packages
         run: |
           npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
-          DIST_TAG=$(cat lerna.json | jq '.version' --raw-output | xargs npm exec -- @k8slens/semver --prerelease 0)
+          VERSION=$(cat lerna.json | jq '.version' --raw-output)
+          echo ${VERSION}
```
Comment (jim-docker) [inline .github/workflows/publish-release-npm.yml:42]: did you mean to leave this? Personally, I think it is a good idea, helps with debugging if there are issues in the future

File: packages/core/src/extensions/common-api/utils.ts (hunk 1 above)
Comment (Nokel81) [reply packages/core/src/extensions/common-api/utils.ts:12]: No they are, I changes the use from `{ Util }` to `* as Util`

Comment (jim-docker) [reply packages/core/src/extensions/common-api/utils.ts:12]: ah right, in line with other types like this.

File: .github/workflows/publish-release-npm.yml (hunk 2 above)
Comment (Nokel81) [reply .github/workflows/publish-release-npm.yml:42]: Yes I did mean to leave this in.

**Review by jakolehm — APPROVED** [review]

**Pull request merged**